### PR TITLE
Fix coding style in crypto/rsa directory

### PR DESCRIPTION
**Pull request by InfoHunter**
There are two commits in this PR:
1. fix the parenthesis in return statements (the return (x) stuffs)
2. fix the braces matching in if-else statements.

The second commit is going to be squashed, I think.

### CODE REVIEW

File: crypto/rsa/rsa_crpt.c [hunk 1]
```diff
@@ -17,41 +17,41 @@
...
                        RSA *rsa, int padding)
 {
-    return (rsa->meth->rsa_pub_dec(flen, from, to, rsa, padding));
+    return rsa->meth->rsa_pub_dec(flen, from, to, rsa, padding);
 }
 
 int RSA_flags(const RSA *r)
 {
-    return ((r == NULL) ? 0 : r->meth->flags);
+    return (r == NULL) ? 0 : r->meth->flags;
```
Comment (levitte) [inline crypto/rsa/rsa_crpt.c]: The remaining parens can be removed as well.

File: crypto/rsa/rsa_lib.c [hunk 2]
```diff
@@ -148,17 +149,17 @@ int RSA_up_ref(RSA *r)
 
     REF_PRINT_COUNT("RSA", r);
     REF_ASSERT_ISNT(i < 2);
-    return ((i > 1) ? 1 : 0);
+    return (i > 1) ? 1 : 0;
```
Comment (levitte) [inline crypto/rsa/rsa_lib.c]: Remove remaining parens too

**Comment by InfoHunter** [comment]
New commit pushed to fix nits

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**